### ISSUE TITLE
[1.12.2] Question about the config file

### ISSUE THREAD

**Issue by fkrisi11** [zlainsama/CleanView#10]
Hey there,

Is there a place where the mod stores if particles should be visible or not?
Like a config file or something maybe?

**Comment by zlainsama** [comment]
No, CleanView does not have a config file.
It only has a key to toggle when playing.

**Comment by fkrisi11** [comment]
So it doesn't store a "toggled" value, and gets reset when you restart the game?

**Comment by zlainsama** [comment]
Yes, as the mod's sole purpose is to hide those particles, storing an off state does not make sense to me.

**Comment by fkrisi11** [comment]
I see. I just wanted to have some insight on this. You are right, it doesn't make any sense to store it.

That's all I wanted to know.
Thank you for the info ^^
